vmmake

```diff
@@ -120,7 +121,11 @@ IPADDR=
 PARTSIZE=10g
 PACKAGES=
 TMPFS=
-while getopts a:i:p:s:t o; do
+KERNCONFIG=
+NUMFILES=
+MKSRC=0
+NONET=0
+while getopts a:i:p:s:tc:f:kx o; do
```

Comment:
Nit: please keep these sorted. Ditto in the usage and switch statement for the options.